test/e2e/node/runtimeclass.go

```diff
@@ -92,54 +81,6 @@ var _ = SIGDescribe("RuntimeClass [Feature:RuntimeClass]", func() {
 		expectSandboxFailureEvent(f, pod, fmt.Sprintf("\"%s\" not found", rcName))
 	})
 
-	It("should recover when the RuntimeClass CRD is deleted [Slow]", func() {
```

Comment:
It's no longer a CRD, so the test doesn't really make sense. I suppose we could keep it for the transition from CRD to internal, but maybe it would be better to just add an upgrade test in that case...

Reply:
The test title says `RuntimeClass CRD is deleted` and so I thought even the new core object could be deleted which would mean the test is still valid, Are you saying that the test servers a different purpose?

Reply:
This was testing that the CRD itself (kind CustomResourceDefinition) was deleted, not that a RuntimeClass object was deleted. APIs can be disabled in the apiserver, but testing that would require some more specialized test infrastructure (since the apiserver would need to be restarted).